lib/vm.js

```diff
@@ -54,4 +54,11 @@ exports.runInThisContext = function(code, options) {
   return script.runInThisContext(options);
 };
 
+exports.runInGlobalContext = function(code, options) {
+  var script = new Script(require('module').wrap(code), options);
+  return script.runInThisContext(options)(exports, require, module,
+                                                            __filename,
+                                                            __dirname);
```

Comment:
It's the vm module's `module` object (as is `exports`).  This function is essentially `vm.runInVmContext()`...